package.json

```diff
@@ -29,6 +29,7 @@
         "@ngx-translate/http-loader": "6.0.0",
         "@popperjs/core": "2.10.2",
         "@sentry/browser": "6.13.3",
+        "@swimlane/ngx-charts": "19.1.0",
```

Comment:
`ngx-charts` is already defined on line 58

Reply:
Strange that it is duplicated. I'll remove it

Reply:
removed in fef44a9